code/modules/admin/verbs/randomverbs.dm

```diff
@@ -983,3 +983,11 @@ var/list/datum/outfit/custom_outfits = list() //Admin created outfits
 	message_admins("[key_name_admin(usr)] toggled their admin antag HUD [adding_hud ? "ON" : "OFF"].")
 	log_admin("[key_name(usr)] toggled their admin antag HUD [adding_hud ? "ON" : "OFF"].")
 	feedback_add_details("admin_verb","TAH") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+
+/client/proc/open_shuttle_manipulator()
+	set category = "Admin"
+	set name = "Shuttle Manipulator"
+	set desc = "Opens the shuttle manipulator UI."
+
+	for(var/obj/machinery/shuttle_manipulator/M in machines)
+		M.ui_interact(usr)

```

Comment:
Nah, it'll just open them all. Also, the shuttle manipulator is an empty machine, it only stores UI state related stuff. It's a machine because I thought it was cool, not because it has to be. Turns out you can attach TGUI to datums.